src/Tester/Tester.csproj

```diff
@@ -74,8 +90,9 @@
     <Reference Include="Microsoft.WindowsAzure.Storage">
       <HintPath>$(SolutionDir)packages\WindowsAzure.Storage.4.2.0\lib\net40\Microsoft.WindowsAzure.Storage.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.5.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
```

Comment:
Don't use fully versioned assembly references in project files. 

The hint path should be all that is required for build to find the package version it should use.